### PR TITLE
Move Firebase to Master

### PR DESCRIPTION
**Pull request by AkuaQ**
With Firebase installed, the movement of all logic will move to the dev_pod. User logic is first to move.

### CODE REVIEW

**Comment by NathanDVT** [comment]
Are you sure you want to push a feature to master

File: Dev_Pod.podspec [hunk 1]
```diff
@@ -41,7 +41,8 @@ This app searches through a collection of recipes of food inside of your kitchen
   s.static_framework = true
   s.dependency 'Firebase'
   s.dependency 'SwiftLint'
+  s.dependency 'Firebase/Core'
   s.dependency 'Firebase/Analytics'
   s.dependency 'Firebase/Auth'
-  s.dependency 'Firebase/Database'
```
Comment (NathanDVT) [inline Dev_Pod.podspec:46]: Why did you remove Database is it depreciated or what?

Comment (AkuaQ) [reply Dev_Pod.podspec:46]: I don't use it

**Review by NathanDVT — APPROVED** [review]
LGTM
